### PR TITLE
Fixing google chrome installation

### PR DESCRIPTION
**Pull request by wlayzz**
I Split the installation of Ghunt and google-chrome. Ghunt is still commented and on FIXME status.

### CODE REVIEW

File: sources/install.sh [hunk 1]
```diff
@@ -1337,15 +1337,15 @@ function install_pwnedornot() {
   add-test-command "pwnedornot.py --help"
 }
 
+function install_chromium() {
+  apt-get install -y chromium
```
Comment (QU35T-code) [inline sources/install.sh:1341]: You need to use `fapt` instead of `apt-get install -y` -> `fapt chromium`

File: sources/install.sh [hunk 2]
```diff
@@ -3326,6 +3326,7 @@ function package_base() {
   install_exegol-history
   install_logrotate
   fapt openjdk-17-jre
+  install_chrome
```
Comment (QU35T-code) [inline sources/install.sh:3329]: Here the wrong function name is used. It's more like `install_chromium`

**Review by QU35T-code — CHANGES_REQUESTED** [review]
Hi,

Thanks for the PR !
Here are some tips to make it good :)

File: sources/install.sh (hunk 1 above)
Comment (wlayzz) [reply sources/install.sh]: Yes thank you, done !

File: sources/install.sh (hunk 2 above)
Comment (wlayzz) [reply sources/install.sh]: Done

**Review by QU35T-code — APPROVED** [review]
Everything is good for me

File: sources/install.sh [hunk 3]
```diff
@@ -1337,15 +1337,15 @@ function install_pwnedornot() {
   add-test-command "pwnedornot.py --help"
 }
 
+function install_chromium() {
+  fapt chromium
+  add-test-command "chromium --version"
+}
+
 function install_ghunt() {
```
Comment (Dramelac) [inline sources/install.sh:1345]: Could you find a test command for ghunt ? To validate the chromium dependencies etc

**Review by Dramelac — CHANGES_REQUESTED** [review]

File: sources/install.sh (hunk 3 above)
Comment (wlayzz) [reply sources/install.sh:1346]: The installation of Ghunt is commented as this tool has problems but I added a test-command for gowitness to validate the chromium dependencies

Comment (Dramelac) [reply sources/install.sh:1346]: Maybe add a FIXME comment then

Comment (wlayzz) [reply sources/install.sh:1347]: ok it's done

**Review by Dramelac — APPROVED** [review]

**Comment by Dramelac** [comment]
Seems good to me.
Wait for CI/CD pipeline to complete and validate every test before merging.

**Comment by Dramelac** [comment]
@ShutdownRepo you can merge it if you want to add this fix to the 3.0.1 release.

**Pull request merged**